Desktop.css

```diff
@@ -182,7 +183,7 @@ main {
   top: 850px;
 }
 
-.work {
+.works {
```

Comment:
- [ ] Good job styling the modal, but your design does not match the original design guidelines. Please kindly ensure that your design matches the original design guidelines(color, typography, spacing, layout etc)
- [ ] Also, the modal is not responsive on the mobile because it has a fixed with. Please kindly ensure it is responsive on both desktop and mobile(different screens)

### Your Design
![image](https://user-images.githubusercontent.com/43995830/210798710-b37fa62d-9993-41ce-a47b-488f5c4165f7.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/210798869-d9127bca-2b1f-4347-926b-23d9426e69dd.png)

Reply:
In case, you don't see my partner as participant, it will not be my fault. his commits are still there. buy somehow github is not showing his profile. Thank you for understanding

Reply:
- [ ] Awesome working on the required changes by the previous reviewer, please I noticed that the pop-up for the desktop version is not displayed correctly according to the design. The two button elements are outside the parent container. Kindly fix it. 👍 
- [ ] Please ensure that the correct content is popped-up when `See project` is triggered.
- [ ] You may need to align the design according to the template. 👍 

## Your Design
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text-removebg-preview (1)](https://user-images.githubusercontent.com/103386183/211027396-ae0e6cc4-7430-40cd-949a-aeb40f97a0be.png)